README.md

```diff
@@ -49,7 +49,16 @@ this may prompt you for sudo access.
     grunt install
     grunt uninstall
 
-# Setting Up a Virtual Machine
+# Setting Up a Physical or Virtual Machine Without Using Vagrant
+
+The next section describes an automated deployment approach that involves using Vagrant. If the GPII Linux Framework needs to be set up on a physical or virtual machine without using Vagrant then the ``install.sh`` script in the ``provisioning`` directory can be used. The script will bootstrap the environment to the point where Ansible roles can be used to set up the Framework. You will need administrative privileges (via sudo) and GNOME 3 set up on Fedora 23 or 24 before using the script:
+```
+cd provisioning
+sudo GPII_FRAMEWORK_DIR=/opt/gpii-linux-framework ./install.sh
```

Comment:
This command won't work unless you either "chmod +x install.sh" or call "sh ./install.sh"